docs/workflows/sdk-setup.md

```diff
@@ -0,0 +1,291 @@
+
+:::tip Sample App
+Sample apps are available to explore. Check out [Example](https://github.com/gobeyondidentity/bi-sdk-react-native/tree/main/example) for the Embedded SDK.
+:::
+
+</TabItem>
+</Tabs>
+
+## Installation
+
+<Tabs groupId="sdks">
+<TabItem value="android" label="Android">
+
+### Gradle
+
+To enable the retrieval of Cloudsmith hosted packages via Gradle, we need to add the Cloudsmith repository to
+the `root/build.gradle` file.
+
+```groovy
+repositories {
+    maven {
+        url "https://packages.beyondidentity.com/public/bi-sdk-android/maven/"
+    }
+}
+```
+
+After the repository is added, we can specify the Beyond Identity dependencies.
+
+```groovy
+dependencies {
+    implementation 'com.beyondidentity.android.sdk:embedded:[version]'
+}
+```
+
+</TabItem>
+<TabItem value="ios" label="iOS">
+
+### Swift Package Manager
+
+#### From Xcode
+
+1. From the Xcode `File` menu, select `Add Packages` and add the following url:
+
+```
+https://github.com/gobeyondidentity/bi-sdk-swift
+```
+
+2. Select a version and hit Next.
+3. Select a target matching the SDK you wish to use.
+
+#### From Package.swift
+
+1. With [Swift Package Manager](https://swift.org/package-manager),
+   add the following `dependency` to your `Package.swift`:
+
+```javascript
+dependencies: [
+    .package(url: "https://github.com/gobeyondidentity/bi-sdk-swift.git", from: [version])
+]
+```
+
+2. Run `swift build`
+
+### Cocoapods
+
+Add the pod to your Podfile:
+
+```javascript
+pod 'BeyondIdentityEmbedded'
+```
+
+And then run:
+```javascript
+pod install
+```
+
+After installing import with
+```javascript
+import BeyondIdentityEmbedded
+```
+
+</TabItem>
+<TabItem value="javascript" label="JavaScript">
+
+```
+yarn add @beyondidentity/bi-sdk-js
+```
+or
+```
+npm install @beyondidentity/bi-sdk-js
+```
+
+</TabItem>
+<TabItem value="flutter" label="Flutter">
+
+### Pub.Dev
+
+Add the Beyond Identity Embedded SDK to your dependencies
+
+```
+dependencies:
+  bi_sdk_flutter: x.y.z
+```
+
+and run an implicit `flutter pub get`.
+
+### Update Android
+
+Please make sure your `android/build.gradle` supports `minSdkVersion` 26 or later.
+
+```
+buildscript {
+  ext {
+    minSdkVersion = 26
+  }
+}
+```
+
+### Update iOS
+
+Please make sure your project supports "minimum deployment target" 13.0 or later.
+In your `ios/Podfile` set:
+
+```sh
+platform :ios, '13.0'
+```
+
+</TabItem>
+<TabItem value="reactnative" label="React Native">
+
+```javascript
+yarn add @beyondidentity/embedded-react-native
+```
+
+or
+
+```javascript
+npm i --save @beyondidentity/embedded-react-native
+```
+
+:::warning
+Note that this library is not compatible with Expo projects. This is because Expo's “managed” workflow does not allow usage of React Native libraries that introduce their own native code beyond the React Native APIs and components that are available in the Expo client app. If you have an Expo app and wish to use this project, you must eject.
```

Comment:
This will be outdated for React once we release. I wonder if we should migrate this MR over here? https://github.com/gobeyondidentity/developer-docs/pull/202

Reply:
Yes, I considered that, but I didn't want to include/use other PRs yet in case these 1-pagers rollout before SDK release.